# A vector of the evidence values
int32[] values

# A vector of corresponding evidence IDs, in accordance with the current
# global characteristic description
string[] evidence_ids

# The ID of the active user
uint32 user_id